app-multimedia/yesplaymusic/autobuild/overrides/usr/share/applications/yesplaymusic.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=YesPlayMusic
```

Comment:
Please add `zh_CN` localisation for `Name[zh_CN]`, `Comment[zh_CN]`, and `Categories[zh_CN]`.

When localising `Categories`, list all English categories, then append Chinese localisation.